Datahub.Web/.editorconfig

```diff
@@ -0,0 +1,160 @@
+﻿# editorconfig.org
```

Comment:
Add note to enable .editorconfig in readme.md ? Not enabled by default in some editors.